pkg/operator/controllers/workaround/systemreserved.go

```diff
@@ -26,8 +26,12 @@ type systemreserved struct {
 	versionFixed *version.Version
 }
 
+// Tweaked values from from https://github.com/openshift/kubernetes/blob/master/pkg/kubelet/apis/config/v1beta1/defaults_linux.go
 const (
 	hardEviction                = "500Mi"
+	nodeFsAvailable             = "10%"
+	nodeFsInodes                = "5%"
+	imageFs                     = "15%"
```

Comment:
Probably OK given upstream this has not changed since 2018 but any thoughts on vendoring this in so we get any updates if it changes?

Reply:
It's a lot to vendor in just for this, as MJ said: https://github.com/Azure/ARO-RP/pull/1472#discussion_r625149966

![image](https://user-images.githubusercontent.com/24868312/116925395-2470b400-ac27-11eb-9559-2f2016bf5cd3.png)

Reply:
Next time it will change is when we have dynamic limits and we gonna remove this. Vendor is from master, so we are ok until 1.22? + vendoring half of the kubelet code for 4 lines of code and introducing all its dependencies sounds overkill from code perspective.

Reply:
+1 to as-is, didn't catch earlier comments regarding scope of vendoring.  Thanks.